namer/videophash.py

```diff
@@ -68,45 +66,20 @@ def __generate_image_thumbnail(self, file: Path) -> Optional[Image.Image]:
     def get_stash_phash(self, file: Path) -> Optional[PerceptualHash]:
         return self.__execute_stash_phash(file)
 
-    def copy_resource_to_file(self, full_path: str, output: Path) -> bool:
```

Comment:
my concern is the that this might not work for pips, which is a supported install mechanism.